include/avif/avif.h

```diff
@@ -862,6 +862,10 @@ AVIF_API avifResult avifImageAllocatePlanes(avifImage * image, avifPlanesFlags p
 AVIF_API void avifImageFreePlanes(avifImage * image, avifPlanesFlags planes);           // Ignores already-freed planes
 AVIF_API void avifImageStealPlanes(avifImage * dstImage, avifImage * srcImage, avifPlanesFlags planes);
 
+// Add arbitrary (opaque) properties to the image
```

Comment:
There should be a comment about how to add a `FullBox` (first four bytes of `data`).

Alternatively there could be yet another function with `version` and `flags` arguments but I guess it would be overkill and it would not match the decoding API.

Note that `FullBox`es are described in `avifImageItemProperty` comments but these functions do not use the `avifImageItemProperty` struct.

Reply:
I've added some extra usage notes.